quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/configuration/KeycloakConfigSourceProvider.java

```diff
@@ -63,6 +63,9 @@ public static void reload() {
 
     @Override
     public Iterable<ConfigSource> getConfigSources(ClassLoader forClassLoader) {
+        if(Environment.isTestLaunchMode()) {
```

Comment:
We should be able to remove this check now we have a specific config source for testing purposes.